frontend/src/lib/zenoapi/models/FilterPredicateGroup.ts

```diff
@@ -3,13 +3,12 @@
 /* tslint:disable */
 /* eslint-disable */
 
-import type { FilterPredicate } from './FilterPredicate';
 import type { Join } from './Join';
 
 /**
  * Group of filter predicates that might be joined by a Join operator.
  */
 export type FilterPredicateGroup = {
-	predicates: Array<FilterPredicateGroup | FilterPredicate>;
+	predicates: any[];
```

Comment:
Is this really necessary? Would be much better if this was not any. We're using it in the front end as well...